doc/nrf/libraries/bluetooth_services/services/hrs_client.rst

```diff
@@ -7,43 +7,35 @@ GATT Heart Rate Service (HRS) Client
    :local:
    :depth: 2
 
-You can use the Heart Rate Service Client to retrieve information about heart rate measurements, such as the sensor location from a device that provides a Heart Rate Service.
-The client can also configure the Heart Rate Service on a remote device by writing specific values to the Heart Rate Control Point characteristic.
+The GATT Heart Rate Service Client is used to retrieve information about heart rate measurements.
 
-Library files
-*************
-
-Files used by Heart Rate Service Client.
+Overview
+********
 
-| Header file: :file:`include/bluetooth/services/hrs_client.h`
-| Source file: :file:`subsys/bluetooth/services/hrs_client.c`
-
-Usage
-*****
+The GATT HRS Client can retrieve information such as the sensor location from a device that provides the Heart Rate Service.
+It can also configure the Heart Rate Service on a remote device by writing specific values to the Heart Rate Control Point characteristic.
 
-.. note::
-   Do not use any of the values in the :c:struct:`bt_hrs_client` object structure directly.
-   All values that should be accessed have accessory functions.
-   The structure is fully defined because the application must be able to allocate memory for it.
+Configuration
+*************
 
 Once a connection with a remote device providing a Heart Rate Service is established, the client needs service discovery to discover Heart Rate Service handles.
```

Comment:
How to establish connection?

Reply:
Application that uses the hrs_client library is responsible for connecting to a peripheral that has the HRS service. Typically it will scan the advertising devices (using the scanning module described in the other file in this PR) and if the scanner detects that an advwertiser supports the HRS service, the application will connect to it and it will start receving the Heart Rate data sutin the hrs_client library.

Reply:
I would like to add it here as the first paragraph in the 'Configuration' subsection. Please check if I haven't changed the meaning by accident: 
_Any application that uses the GATT HRS Client library should connect to peripherals having the HRS. The application uses the scanning module to detect advertising devices. If the scanner detects that an advertiser supports the HRS, the application will connect to it automatically and will start receiving HRS data._

Reply:
Yes, I think it looks good :)